lib/tasks/openapi_generate.rake

```diff
@@ -0,0 +1,561 @@
+class OpenapiGenerator
```

Comment:
We will have this in 3 repos now, wonder if we can make this general enough to build a gem out of

Reply:
not sure, we can probably get a general one for Rails. But for this, we scan InventoryCollection objects and models inside of persister schema, so it's kind of specific.